regression/run_regression.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Is it not possible that you use update alternatives and then change the python to whatever is required instead of changing 10 files for that?

Reply:
Instead of adding one more script file and keeping the shebang line as such will not give me any minimization of code as the script will still have the shebang.
Also, since no python2 specific code , I guess this is simpler way to proceed.